css-fonts/css-fonts/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Fonts</title>
+  <link href="styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Neonderthaw&family=Roboto:wght@100&display=swap" rel="stylesheet">
```

Comment:
It's generally good practice to place your custom stylesheet `<link>`s after other pre-made `<link>`s. Otherwise, the fonts may not be ready to go, and library stylesheets may take precedence over your custom one.